pkg/action/install.go

```diff
@@ -13,6 +13,9 @@ type Install struct {
 	Driver driver.Driver // Needs to be more than a string
 }
 
+// makes sure Install implements Action interface
+var _ Action = &Install{}
```

Comment:
I think we used to put this check in `_test.go` files, but I'm not sure this is consistent across the entire code base.

Reply:
I moved them to their respective test files 👍